tensorflow/lite/micro/tools/make/targets/bluepill_makefile.inc

```diff
@@ -66,8 +66,7 @@ EXCLUDED_TESTS := \
   tensorflow/lite/micro/micro_interpreter_test.cc \
   tensorflow/lite/micro/micro_allocator_test.cc \
   tensorflow/lite/micro/memory_helpers_test.cc \
-  tensorflow/lite/micro/memory_arena_threshold_test.cc \
-  tensorflow/lite/micro/kernels/circular_buffer_test.cc
+  tensorflow/lite/micro/memory_arena_threshold_test.cc
```

Comment:
let's also enable for the stm32f4 target

Reply:
Done.